servicenow_connector.py

```diff
@@ -1564,33 +1638,38 @@ def _on_poll(self, param):
 
         action_query = config.get(SERVICENOW_JSON_ON_POLL_FILTER, "")
 
-        if (len(action_query) > 0):
-            query += '^' + action_query
+        if len(action_query) > 0:
+            query += '^{}'.format(action_query)
 
         # If it's a poll now don't filter based on update time
         if self.is_poll_now():
             max_tickets = param.get(phantom.APP_JSON_CONTAINER_COUNT)
         # If it's the first poll, don't filter based on update time
-        elif (self._state.get('first_run', True)):
+        elif self._state.get('first_run', True):
             self._state['first_run'] = False
             max_tickets = self._first_run_container
         # If it's scheduled polling add a filter for update time being greater than the last poll time
         else:
             # "last_time" should be of the format "%Y-%m-%d %H:%M:%S"
             if last_time and len(last_time.split(" ")) == 2:
-                query += "^sys_updated_on>=javascript:gs.dateGenerate('{}','{}')".format(last_time.split(" ")[0], last_time.split(" ")[1])
+                query_prefix = last_time.split(" ")
+                query += "^sys_updated_on>=javascript:gs.dateGenerate('{}','{}')".format(query_prefix[0], query_prefix[1])
                 max_tickets = self._max_container
             else:
-                self.debug_print("Either 'last_time' is None or empty or it is not in the expected format of %Y-%m-%d %H:%M:%S. last_time: {}".format(last_time))
-                self.debug_print("Considering this as the first scheduled|interval polling run; skipping time-based query filtering; processing the on_poll workflow accordingly")
+                self.debug_print("Either 'last_time' is None or empty or it is not \
+                    in the expected format of %Y-%m-%d %H:%M:%S. last_time: {}".format(last_time))
+                self.debug_print("Considering this as the first scheduled|interval \
+                    polling run; skipping time-based query filtering; processing the on_poll workflow accordingly")
 
                 max_tickets = self._first_run_container
 
-                self.debug_print("Setting the 'max_tickets' to the value of 'first_run_container'. max_tickets: {}".format(max_tickets))
+                self.debug_print("Setting the 'max_tickets' to the value of \
+                    'first_run_container'. max_tickets: {}".format(max_tickets))
```

Comment:
Do you think we can fit the whole line on the 2nd line without exceeding the lint limit?

Reply:
Yes, I have made the suggested changes.